include/dataformats/trigger/TriggerPrimitivesFragment.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef DATAFORMATS_INCLUDE_DATAFORMATS_TRIGGER_TRIGGERPRIMITIVESFRAGMENT_HPP_
```

Comment:
File header comment missing, including copyright info

Reply:
Fixed in 30f2d64